drlineconfiguration.json

```diff
@@ -1044,35 +1138,53 @@
     "name": "SABIO-RK",
     "displayName": "SABIO-RK",
     "category": "EAP",
-    "uriLink": "http://sabiork.h-its.org/newSearch?q=UniProtKB_AC:%u"
+    "uriLink": "http://sabiork.h-its.org/newSearch?q=UniProtKB_AC:%id"
   },
   {
     "name": "SignaLink",
     "displayName": "SignaLink",
     "category": "EAP",
-    "uriLink": "http://signalink.org/protein/%s"
+    "uriLink": "http://signalink.org/protein/%id"
   },
   {
     "name": "SIGNOR",
     "displayName": "SIGNOR",
     "category": "EAP",
-    "uriLink": "https://signor.uniroma2.it/relation_result.php?id=%u"
+    "uriLink": "https://signor.uniroma2.it/relation_result.php?id=%id"
   },
   {
     "name": "UniPathway",
     "displayName": "UniPathway",
     "category": "EAP",
-    "uriLink": "http://www.unipathway.org?upid=%s&entryac=%u",
+    "uriLink": "http://www.unipathway.org?upid=%id&entryac=%accession",
     "attributes": [{
       "name": "RectionId",
       "xmlTag": "reaction ID"
     }]
   },
+  {
+    "name": "PlantReactome",

```

Comment:
Unable to test this as there is no `databaseCrossReferences` entry for this:  https://wwwdev.ebi.ac.uk/uniprot/api/uniprotkb/accession/Q0DBF4 while it is present in: https://www.uniprot.org/uniprot/Q0DBF4